### ISSUE TITLE
Plugin ODT and plugin Bookcreator show broken/strange icons

### ISSUE THREAD

**Issue by LarsGit223** [ipari/dokuwiki-template-white#9]
That plugins show broken and strange behaving icons in the pagetools. This seems to be because they use their own CSS and are not handled in ```white_toolsevent()``` to receive icons from template white.

Remark:
The plugins might add more than one button per plugin.

**Comment by LarsGit223** [comment]
So it would be nice to add them for ODT or bookcreator users.

It might be a good idea to disable the icons from unknown/unhandled plugins by assigning them some default class (e.g. something like ```plugin_no_icon```). Today that would have "only" caused a missing icon for the ODT and bookcreator plugin. But it still looks better than an unpredictable behaviour because you never know if the CSS/icon of a plugin fits to your template. Just an idea.

**Comment by ipari** [comment]
Fixed this issue on d3528304ddabc4e6f46131340206533469946743. Thanks :)

**Comment by LarsGit223** [comment]
Thanks for the fix. The icons are not messed up any more.

But there is still a little issue: I don't see text beside the ODT plugin icons.
See this example screenshot:

![templatewhiteiconsnotext](https://cloud.githubusercontent.com/assets/9009011/23822179/61232e30-0647-11e7-912c-c8c7a80cbe00.png)

So, all other texts are there. Just the text from the ODT plugin icons seems to be missing.

**Comment by ipari** [comment]
OK I'll check it

**Comment by LarsGit223** [comment]
The second button from the ODT plugin is only shown if activated in the configuration. On default it's deactive. Please check the configuration of the ODT plugin and activate option ```showpdfexportbutton```.

**Comment by LarsGit223** [comment]
I checked what happens if I only activate one of the buttons. Same result. Icon is OK but no text. By the way if I hover the mouse over the menu item/icon then the tool tip shows the text "ODT export". That is ok but no text besides the icon.

**Comment by ipari** [comment]
It's hard to guess what cause this issue. I tested on last two version of DokuWiki in two browsers(Chrome and IE), but shows both icon and text in all of them. Can you give me more information?

## Release 2017-02-19b "Frusterick Manners"
### Chrome
![image](https://cloud.githubusercontent.com/assets/3617005/23823500/60bcdd46-06a7-11e7-9792-ac75334f9444.png)

### IE
![image](https://cloud.githubusercontent.com/assets/3617005/23823523/a3cb34e8-06a7-11e7-8020-c6a7bea4dbe4.png)


## Release 2016-06-26a "Elenor of Tsort"
### Chrome
![image](https://cloud.githubusercontent.com/assets/3617005/23823503/65ee0ad8-06a7-11e7-9cf5-74fc08038a07.png)

### IE
![image](https://cloud.githubusercontent.com/assets/3617005/23823515/8feb7a28-06a7-11e7-87c8-50e806f12d9d.png)

**Comment by LarsGit223** [comment]
I have got Release 2017-02-19a "Frusterick Manners", Firefox on an ubuntu machine.

**Comment by LarsGit223** [comment]
I found the problem. I have done some experiments on my installation and the ODT plugin. I forgot to restore some code for the ODT plugin buttons.

It was like this:
```
$this->getLang('export_odt_button')
```
and I restored it to:
```
'<span>' . $this->getLang('export_odt_button') . '</span>'
```
So the missing spans caused the problems. Of course you did not have this problems with the official ODT plugin release.

Sorry for wasting your time. My fault :worried:
